test/play_scraper_test.py

```diff
@@ -0,0 +1,80 @@
+from pathlib import Path
+from urllib.error import HTTPError, URLError
+import numpy as np
+import pytest
+from src import PlayScrapingError, PlayPageScraper
+
+ICON_HTML = """
+<img src="%s" class="T75of sHb2Xb">
+"""
+
+CATEGORY_HTML = """
+<a itemprop="genre">Example</a>
+"""
+
+SIMPLE_HTML = """
+<p>Simple paragraph.</p>
+"""
+
+ICON_SUBDIR = Path("icon_subdir")
+
+
+class TestPlayPageScraper:
+    def test_get_icon(self, test_dir, icon_dir, play_scraper):
+        rand_array = np.random.randint(0, 255, 10)
+        icon_name = "icon_com.example"
+        np.savetxt(test_dir / icon_name, rand_array, fmt="%d")
+
+        html_file = test_dir / "com.example"
+        html_img_src = pytest.FILE + str(test_dir / icon_name)
+        html_file.write_text(ICON_HTML % html_img_src)
+
+        play_scraper.get_icon("com.example", ICON_SUBDIR)
+
+        read_array = np.loadtxt(icon_dir / ICON_SUBDIR / icon_name, dtype=int)
+        assert np.array_equal(rand_array, read_array)
+
+    def test_get_category(self, play_scraper, test_dir):
+        expected_genre = "example"
+
+        html_file = test_dir / "com.example.bis"
```

Comment:
If you don't reuse your test dir (by not setting a scope), those filenames can always be the same, making your tests slightly easier to read.